src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/Html/DefaultDisplayTemplates.cs

```diff
@@ -237,23 +234,16 @@ public static IHtmlContent ObjectTemplate(IHtmlHelper htmlHelper)
                     continue;
                 }
 
-                var divTag = new TagBuilder("div");
-
                 if (!propertyMetadata.HideSurroundingHtml)
                 {
                     var label = propertyMetadata.GetDisplayName();
                     if (!string.IsNullOrEmpty(label))
                     {
-                        divTag.SetInnerText(label);
-                        divTag.AddCssClass("display-label");
-                        content.AppendLine(divTag.ToHtmlContent(TagRenderMode.Normal));
-
-                        // Reset divTag for reuse.
-                        divTag.Attributes.Clear();
-                    }
-
-                    divTag.AddCssClass("display-field");
-                    content.Append(divTag.ToHtmlContent(TagRenderMode.StartTag));
+                        var labelTag = new TagBuilder("div");
```

Comment:
@Eilon at least for me `labelTag` is confusing because this builder will not generate a `<label>`. That name doesn't say anything about what the builder will do, just what the generated body will be.

Would `labelDivBuilder` (or perhaps `nameDivBuilder` since this is a display template) work here and `valueDivBuilder` just below? If not what names do you suggest?